src/actions/transcode/StreamingProfile.ts

```diff
@@ -0,0 +1,18 @@
+import Action from "../Action";
+import Param from "../../parameters/Param";
+
+class StreamingProfileAction extends Action {
+  private profile: string|number;
+
+  constructor(profile: string|number) {
+    super();
+    this.profile = profile;
+  }
+
+  protected prepareParam(): this {
+    this.addParam(new Param('sp', this.profile));
```

Comment:
shouldn't this just be in the constructor?
Do we need it in the prepare param?

Reply:
updated